MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include versioneer.py
+include lib/nc_time_axis/_version.py
```

Comment:
But it does keep the package cleanly isolated so that if you use a `.pth` file to point to the package, then your not exposed to any Python files in the top level directory e.g. `setup.py`